src/olympia/shelves/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+from olympia.amo.models import ModelBase
+
+CHOICE_TYPES = (
+    ('category', 'category'),
+    ('collection', 'collection'),
+    ('extension', 'extension'),
+    ('recommended', 'recommended'),
+    ('search', 'search'),
+    ('theme', 'theme'),
+)
```

Comment:
Mmm, we might need this list elsewhere, so instead lets have a flat tuple of the shelf types `SHELF_TYPES = ('category', 'collection', ...)` and then `SHELF_TYPE_CHOICES = tuple((ty, ty) for ty in SHELF_TYPES)` for the field choices